cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,28 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        it('plays media when a user clicks play', () => {
+          cy.window().then(win => {
+            const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+            if (media && media.type === 'video') {
+              cy.get(
+                'div[class^="StyledVideoContainer"] img[class^="StyledImg"]',
+              )
+                .click()
+                .should('not.exist')
+                .then(() => {
+                  cy.get('iframe[class^="StyledIframe"]').then($iframe => {
+                    cy.wrap($iframe.prop('contentWindow'))
+                      .its(
+                        'embeddedMedia.playerInstances.mediaPlayer._currentTime',
```

Comment:
Any member beginning `_` shouts "private API" to me. Is there nothing else we could hook into that looks more like a supported feature?

Reply:
I agree. This put me off to begin with, but as I mentioned in the body, it's reliable. Two points:

- I'm exploring using Bump events as an alternative
- I've left a message with a past iPlayer engineer to see what he thinks about this solution